docs/quickstart.md

```diff
@@ -234,7 +234,8 @@ feature_vector = store.get_online_features(
         'driver_hourly_stats:acc_rate',
         'driver_hourly_stats:avg_daily_trips'
     ],
-    entity_rows=[{"driver_id": 1001}]
+    entity_rows=[{"driver_id": 1001}],
+    full_feature_names=True
```

Comment:
The quickstart should be as simple as possible. Is there a reason to include this argument?

Reply:
I was under the impression that it aims to show all the features in as simple a way as possible (so the ability to prefix feature names with their respective feature views by setting `full_feature_names=True`). But if the idea is to keep it as simple as possible I can remove the full_feature_names argument from quickstart and the README.

Reply:
The idea is to show the core functionality in as simple a way as possible, not all functionality.